ccmlib/scylla_node.py

```diff
@@ -250,7 +251,23 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
                 node.watch_log_for_alive(self, from_mark=mark)
 
         if wait_for_binary_proto:
-            self.wait_for_binary_interface(from_mark=self.mark, process=self._process_scylla)
+            try:
+                self.wait_for_binary_interface(from_mark=self.mark, process=self._process_scylla)
+            except TimeoutError as e:
+                if not self.grep_log('storage_service - JOINING: getting bootstrap token', from_mark=self.mark):
```

Comment:
why not move this logic into it's own function ? `wait_for_bootstrap_repair`  ? I'm guessing we'll have more place it would be needed...

Reply:
we can do it when needed, but no problem making it a method in this change

Reply:
I think it would be a bit more clearer, the start/stop functions already has lots of logic in them already.